src/store/peer/types.ts

```diff
@@ -1,9 +1,23 @@
+import {Group} from "../group/types";
+
 export interface Peer {
+  ID?: string,
   Name: string,
   IP: string,
   Connected: boolean,
   LastSeen: string,
   OS: string,
   Version: string,
-  Groups?: any[]
+  Groups?: Group[]
+}
+
+export interface PeerToSave extends Peer {
+  groupsToSave: string[]
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  groupsToSave: string[];
```